src/main.js

```diff
@@ -80,4 +74,19 @@ document.querySelector(`.trip-main__event-add-btn`).addEventListener(`click`, (e
   tripBoardPresenter.createTask();
 });
 
+api.getData()
+  .then((serverData) => {
```

Comment:
Здесь можно было сразу "распарсить" ответ
```suggestion
  .then(({ offers, destinations, points }) => {
```